<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lasercutter</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="codeTab">
       <attribute name="title">
        <string>Steuercode</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="2" colspan="2">
         <widget class="QPushButton" name="openButton">
          <property name="maximumSize">
           <size>
            <width>320</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Ã–ffnen</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label1">
          <property name="text">
           <string>Steuercode:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QLabel" name="label2">
          <property name="maximumSize">
           <size>
            <width>320</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Datei:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QPushButton" name="saveButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>320</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Speichern</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2" colspan="2">
         <widget class="QLabel" name="label3">
          <property name="maximumSize">
           <size>
            <width>320</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Wertangaben:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QRadioButton" name="relativeRadioButton">
          <property name="maximumSize">
           <size>
            <width>155</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Relativ</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QRadioButton" name="absoluteRadioButton">
          <property name="maximumSize">
           <size>
            <width>155</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Absolut</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="2" colspan="2">
         <widget class="QLabel" name="label4">
          <property name="maximumSize">
           <size>
            <width>320</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Fehlermeldungen:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2" colspan="2">
         <widget class="QPlainTextEdit" name="codeLog">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>320</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Courier</family>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string>Keine Fehler.
Code in Ordnung.
----------------
</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="6" colspan="2">
         <widget class="QTextEdit" name="codeEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Courier</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::WidgetWidth</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="simTab">
       <attribute name="title">
        <string>Lasersteuerung</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="1" colspan="2">
         <widget class="QGroupBox" name="statusGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Status</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="5" column="0">
            <widget class="QLabel" name="commandLineLabel">
             <property name="text">
              <string>Line 7:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label9">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Aktueller Befehl:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label8">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Laserposition:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="laserMoveStateLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Stand</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label7">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Laserstatus:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="commandLabel">
             <property name="text">
              <string>MOVE 150, 100</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="laserPosYLabel">
             <property name="text">
              <string>500</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="laserPosXLabel">
             <property name="text">
              <string>300</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="laserOnStateLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Aus</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>23</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0" rowspan="4">
         <widget class="QGraphicsView" name="simGView"/>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QGroupBox" name="autoGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Automatik</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="2" column="1">
            <widget class="QLabel" name="cmdCountLabel">
             <property name="text">
              <string>19</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QPushButton" name="runCodeButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Steuercode fahren</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="codeSizeLabel">
             <property name="text">
              <string>300 x 500</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label5">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>GrÃ¶ÃŸe:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label6">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Anzahl Befehle:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="codeLogLabel">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Steuercode OK!</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QPushButton" name="stopCodeButton">
             <property name="text">
              <string>AusfÃ¼hrung stoppen</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>openButton</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>relativeRadioButton</tabstop>
  <tabstop>absoluteRadioButton</tabstop>
  <tabstop>codeLog</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
